DocumentsFromSnapshotMigration/src/main/java/org/opensearch/migrations/RfsMigrateDocuments.java

```diff
@@ -181,6 +181,12 @@ public static class Args {
             description = ("Version of the source cluster."))
         public Version sourceVersion = Version.fromString("ES 7.10");
 
+        @Parameter(required = true,
```

Comment:
Target version detection build into the OpenSearch client, can we use that instead of requiring a parameter?

- https://github.com/opensearch-project/opensearch-migrations/pull/1118

Reply:
Easy to do, thanks for the idea!  I'd forgotten about this.